en/components/overlay_position.md

```diff
@@ -7,6 +7,54 @@ _description: Explanation and example about the Overlay Service's IPositionStrat
 
 Position strategies determine where the content is displayed in the provided `IgxOverlayService`. By default, the content is positioned in the middle of the screen.
 
+### Strategies Overview
+
+There are five positioning strategies:
+
+#### Global
+Positions the content, based on the directions passed in through [`positionSettings`]({environment:angularApiUrl}/interfaces/positionsettings.html). These are Left/Center/Right for [`horizontalDirection`]({environment:angularApiUrl}/interfaces/positionsettings.html#horizontaldirection) and Top/Middle/Bottom for [`verticalDirection`]({environment:angularApiUrl}/interfaces/positionsettings.html#verticaldirection). Defaults are:
+
+    | horizontalDirection        | verticalDirection        |
```

Comment:
The rendering of all markdown tables in this section is now borked.

![image](https://user-images.githubusercontent.com/4118303/78893576-3e67b380-7a74-11ea-8838-3130643e43f7.png)